lib/states/az/mesa.R

```diff
@@ -8,39 +8,79 @@ source("common.R")
 # a little unclear as to which charges are specifically pedestrian vs.
 # vehicular, so our categorization here is weak; see outstanding TODO
 load_raw <- function(raw_data_dir, n_max) {
-  d <- load_single_file(
+  old_d <- load_single_file(
```

Comment:
Would be good to add a short comment clarifying what years the old data covered, who processed it, important differences from the new data, etc.